config/application.rb

```diff
@@ -22,5 +22,8 @@ class Application < Rails::Application
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
     config.active_job.queue_adapter = :sidekiq
+
+    config.autoload_paths += %W(#{config.root}/lib)
```

Comment:
If we add `lib` to the `autoload_path`, it might be needed to append it as well to `eager_load_path` (Rails 5 thing)